com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/DeleteServicesStep.java

```diff
@@ -99,129 +87,33 @@ protected String getStepErrorMessageAdditionalDescription(DelegateExecution cont
         return ExceptionMessageTailMapper.map(configuration, CloudComponents.SERVICE_BROKERS, offering);
     }
 
-    private List<CloudServiceExtended> getServicesData(List<String> serviceNames, ExecutionWrapper execution) {
-        CloudControllerClient client = execution.getControllerClient();
+    private CloudServiceInstance getServiceInstanceData(String serviceToDelete, CloudControllerClient client) {

```

Comment:
Why does this method have the suffix 'Data'? Also, the variables assigned in the above methods also include the 'Data' suffix. Do we really need this method anyway, why not just call the client directly where this method is used?